uber/templates/base.html

```diff
@@ -88,15 +88,6 @@
             });
         </script>
         {% endif %}
-      <script type="text/javascript">
```

Comment:
Some javascript inside form fields was overriding this javascript, unhiding empty/blank fields, so I moved it downward to have greater precedence.